crypto/sr25519/private_test.go

```diff
@@ -0,0 +1,133 @@
+package sr25519
+
+import (
+	"reflect"
+	"testing"
+
+	"github.com/mailchain/mailchain/crypto"
+	"github.com/mailchian/mailchain/crypto/sr25519"
```

Comment:
`"github.com/mailchian/mailchain/crypto/sr25519"` should be removed
Looking in the files for `sr25519.XXXXX` and remove `sr25519.` it is part of the same package